src/core/Table/Table.tsx

```diff
@@ -134,6 +140,11 @@ export type TableProps<
    * Must be memoized.
    */
   rowProps?: (row: Row<T>) => React.ComponentPropsWithRef<'div'>;
+  /**
+   * Flag whether show sub-rows or not.
+   * @default false
+   */
+  showSubRows?: boolean;
```

Comment:
Previously I was checking whether there are any rows with `subRows` but I thought people might want to have lazy-loading and first rows might not have any `subRows` and if they appear later it would cause layout shift.

Reply:
If lazy-loading is also used e.g. first 10 rows won't have sub-rows, then another batch of 10 rows are loaded that has sub-rows, then previous rows 10 would shift to the right to match the alignment.

Reply:
Ah ok, horizontal shift.

Can we at least make the default true (more useful and common)? It seems wrong that everyone needs to manually set this to true (just for the few _potential_ cases of lazy loading where first set doesn't have subrows).

Reply:
I dont think it should default to true. Its not that often case to show subrows at all :/